### PR TITLE
Expose calico metrics

### PR DESCRIPTION
**Pull request by urbanserj**
Konvoy PR: https://github.com/mesosphere/konvoy/pull/665

### CODE REVIEW

File: templates/prometheus.yaml [hunk 1]
```diff
@@ -80,6 +80,38 @@ spec:
                   regex: '(.*):10250'
                   replacement: '${1}:1338'
                   target_label: __address__
+            - job_name: 'kubernetes-calico-node'
```
Comment (hectorj2f) [inline templates/prometheus.yaml]: can you share an example of the prometheus targets to validate that this job is currently working as expected ?

Comment (urbanserj) [reply templates/prometheus.yaml]:
```
up{job="kubernetes-calico-node"}


up{endpoint="bird-metrics",instance="10.0.128.213:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-213.us-west-2.compute.internal",pod="calico-node-rxq45"} | 1
up{endpoint="bird-metrics",instance="10.0.128.4:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-4.us-west-2.compute.internal",pod="calico-node-72s98"} | 1
up{endpoint="bird-metrics",instance="10.0.130.134:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-134.us-west-2.compute.internal",pod="calico-node-hhfvb"} | 1
up{endpoint="bird-metrics",instance="10.0.130.183:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-183.us-west-2.compute.internal",pod="calico-node-q9l2p"} | 1
up{endpoint="bird-metrics",instance="10.0.192.82:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-82.us-west-2.compute.internal",pod="calico-node-gbs7q"} | 1
up{endpoint="bird-metrics",instance="10.0.192.92:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-92.us-west-2.compute.internal",pod="calico-node-766wb"} | 1
up{endpoint="bird-metrics",instance="10.0.194.217:9092",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-194-217.us-west-2.compute.internal",pod="calico-node-xg47c"} | 1
up{endpoint="felix-metrics",instance="10.0.128.213:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-213.us-west-2.compute.internal",pod="calico-node-rxq45"} | 1
up{endpoint="felix-metrics",instance="10.0.128.4:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-128-4.us-west-2.compute.internal",pod="calico-node-72s98"} | 1
up{endpoint="felix-metrics",instance="10.0.130.134:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-134.us-west-2.compute.internal",pod="calico-node-hhfvb"} | 1
up{endpoint="felix-metrics",instance="10.0.130.183:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-130-183.us-west-2.compute.internal",pod="calico-node-q9l2p"} | 1
up{endpoint="felix-metrics",instance="10.0.192.82:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-82.us-west-2.compute.internal",pod="calico-node-gbs7q"} | 1
up{endpoint="felix-metrics",instance="10.0.192.92:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-192-92.us-west-2.compute.internal",pod="calico-node-766wb"} | 1
up{endpoint="felix-metrics",instance="10.0.194.217:9091",job="kubernetes-calico-node",name="calico-node",namespace="kube-system",node="ip-10-0-194-217.us-west-2.compute.internal",pod="calico-node-xg47c"} | 1
```

**Comment by shaneutt** [comment]
@urbanserj please sign commits and then :heavy_check_mark:

**Comment by hectorj2f** [comment]
@shaneutt Let's not merge this PR yet. It is blocked by another Pr

**Comment by hectorj2f** [comment]
@urbanserj please sign your commits otherwise we cannot merge this PR

**Comment by urbanserj** [comment]
@shaneutt @hectorj2f I've signed my commit. Please take a look.

**Pull request merged**